model/authentication.go

```diff
@@ -74,6 +75,16 @@ func (auth *Authentication) ToResponse() *AuthenticationResponse {
 
 func (auth *Authentication) ToInternalResponse() *AuthenticationInternalResponse {
 	resourceID := strconv.FormatInt(auth.ResourceID, 10)
+	var pass string
+	if auth.Password == nil {
+		pass = ""
+	} else {
+		decrypted, err := util.Decrypt(*auth.Password)
+		if err != nil {
+			logger.Log.Errorf("failed to decrypt password: %v", err)
```

Comment:
What do you think about including the authentication ID here, so that we can debug which one is causing problems?

Reply:
I like that - added.